config/clusters/azure.carbonplan/support.values.yaml

```diff
@@ -1,3 +1,17 @@
+prometheusAuth:
+  enabled: true
+
+prometheus:
```

Comment:
There is a CI error about this. There are two separate entries for `prometheus` in this file, where one below configures the resource requests.

Reply:
Thanks for catching it! Fixed! All hail CI!